pkg/log/log.go

```diff
@@ -384,16 +388,45 @@ func matchesPod(pod *corev1.Pod, options *Options) bool {
 }
 
 func matchesContainer(pod *corev1.Pod, container corev1.Container, options *Options) bool {
-	if options == nil || options.ContainerReplica == "" {
-		return true
+	if options != nil && options.ContainerReplica != "" {
+		parts := strings.SplitN(options.ContainerReplica, ".", 3)
+		if len(parts) == 3 {
+			return pod.Name == parts[1] && container.Name == parts[2]
+		} else if len(parts) == 2 {
+			if pod.Labels[applabels.AcornContainerName] != "" {
```

Comment:
Good catch with additional job filtering! Mind adding a small test to catch this behavior?

Reply:
@StrongMonkey I went ahead and added several tests for this `matchesContainer` function since its logic is a bit complex. 👍